runatlantis.io/docs/using-atlantis.md

```diff
@@ -79,6 +79,18 @@ atlantis plan -d dir -- -var foo='bar'
 ```
 If you always need to append a certain flag, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
 
+### Using the -destroy Flag
+
+#### Example
+To perform a destructive plan that will destroy resources in the specified directory, you can use the `-destroy` flag like this:
+
+```bash
+atlantis plan -d dir -- -destroy
```

Comment:
```suggestion
atlantis plan -- -destroy
```

Reply:
There are cases where users may run on a different directory, so I think `atlantis plan -d dir -- -destroy` this could also be useful for new users to have